admin/new.php

```diff
@@ -577,7 +577,16 @@ function gigpress_add() {
 					  	$related_posts_sql .= " ORDER BY p.post_date DESC LIMIT 500";
 
 					  	$entries = $wpdb->get_results($related_posts_sql, ARRAY_A);
-					  	if($entries != FALSE) {
+
+						$entries = apply_filters( 'gigpress_related_post_entries', $entries );
+						/**
+						 * Provides an opportunity to specify in details what's available as related posts
+						 *
+						 * @param array $entries
+						 * @since 2.3.24
+						 */
```

Comment:
```suggestion
						/**
						 * Provides an opportunity to specify in details what's available as related posts
						 *
						 * @param array $entries
						 * @since 2.3.24
						 */
						$entries = apply_filters( 'gigpress_related_post_entries', $entries );
```

Reply:
Nice, good to see this in the next release!

Reply:
Can you give us access to your fork or can you commit this change on your side?